src/test/java/org/sagebionetworks/bridge/spring/controllers/ParticipantFileControllerTest.java

```diff
@@ -0,0 +1,149 @@
+
+    UserSession session;
+    ParticipantFile persisted;
+
+    @Captor
+    ArgumentCaptor<ParticipantFile> fileCaptor;
+
+    @BeforeMethod
+    public void beforeMethod() {
+        MockitoAnnotations.initMocks(this);
+
+        session = new UserSession();
+        session.setAppId(TEST_APP_ID);
+        StudyParticipant participant = new StudyParticipant.Builder().withId("test_user").build();
+        session.setParticipant(participant);
+        doReturn(session).when(controller).getAuthenticatedAndConsentedSession();
+        doReturn(mockRequest).when(controller).request();
+        doReturn(mockResponse).when(controller).response();
+
+        persisted = ParticipantFile.create();
+        persisted.setFileId("file_id");
+        persisted.setUserId("test_user");
+        persisted.setCreatedOn(TestConstants.TIMESTAMP);
+        persisted.setAppId("api");
+        persisted.setMimeType("dummy-type");
+    }
+
+    @Test
+    public void verifyAnnotations() throws Exception {
+        assertCrossOrigin(ParticipantFileController.class);
+        assertGet(ParticipantFileController.class, "getParticipantFile");
+        assertGet(ParticipantFileController.class, "getParticipantFiles");
+        assertPost(ParticipantFileController.class, "createParticipantFile");
+        assertDelete(ParticipantFileController.class, "deleteParticipantFile");
+    }
+
+    @Test
+    public void getParticipantFiles() {
+        ParticipantFile file = ParticipantFile.create();
+        file.setUserId("test_user");
+        file.setFileId("file_id");
+        ForwardCursorPagedResourceList<ParticipantFile> page = new ForwardCursorPagedResourceList<>(
+                ImmutableList.of(file), null);
+        when(mockFileService.getParticipantFiles(eq("test_user"), isNull(), anyInt())).thenReturn(page);
+
+        ForwardCursorPagedResourceList<ParticipantFile> result = controller.getParticipantFiles(null, "5");
+        assertSame(page, result);
+
+        verify(mockFileService).getParticipantFiles(eq("test_user"), isNull(), eq(5));
+    }
+
+    @Test
+    public void getParticipantFile() {
+        when(mockFileService.getParticipantFile(eq("test_user"), eq("file_id"))).thenReturn(persisted);
+        ParticipantFile result = controller.getParticipantFile("file_id");
+        assertSame(result, persisted);
+
+        verify(mockFileService).getParticipantFile("test_user", "file_id");
+        verify(mockResponse).setHeader(eq("Location"), eq(result.getDownloadUrl()));
+    }
+
+    @Test
+    public void createParticipantFile() throws Exception {
+        ParticipantFile newFile = ParticipantFile.create();
+        newFile.setUserId("test_user");
+        newFile.setFileId("file_id");
+        newFile.setMimeType("dummy-type");
+        mockRequestBody(mockRequest, newFile);
+        when(mockFileService.createParticipantFile(any(), any(), any())).thenReturn(newFile);
+        ParticipantFile result = controller.createParticipantFile("file_id");
```

Comment:
We should use a different file ID so we can verify that the controller sets the file ID from the input parameters.